nixos/modules/services/monitoring/prometheus/exporters/bind.nix

```diff
@@ -0,0 +1,62 @@
+{ config, lib, pkgs }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.bind;
+in
+{
+  port = 9119;
+  extraOpts = {
+    bindURI = mkOption {
+      type = types.str;
+      default = "http://localhost:8053/";
+      description = ''
+        HTTP XML API address of an Bind server.
+      '';
+    };
+    bindTimeout = mkOption {
+      type = types.str;
+      default = "10s";
+      description = ''
+        Timeout for trying to get stats from Bind.
+      '';
+    };
+    bindPidFile = mkOption {
+      type = types.path;
+      default = "/run/named/named.pid";
+      description = ''
+        Path to Bind's pid file to export process information.
```

Comment:
I don't see a reason to have an option for this.

Reply:
This value is hardcoded in the bind module but a completely differrent configuration can be passed to bind so it could be useful to have this option in this case

Reply:
I'm not convinced, this options seems like an implementation detail nobody really needs in their nixos config. If somebody needs it they can pass it with `extraFlags`

Reply:
This option is used by the exporter to collect some information about the process: https://github.com/digitalocean/bind_exporter/blob/master/bind_exporter.go#L463

https://github.com/prometheus/client_golang/commit/dbd48d666b7cab8fdc1c5fea89ad1db67f79ad03

So you think this option should be hardcoded in the module implementation ? Or not used at all ? And in the last case the user should pass it to `extraFlags` ?